src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1384,6 +1385,14 @@ public void sendSignChange(@NotNull Location loc, @Nullable String @Nullable []
 		}
 	}
 
+	@Override
+	public void sendBlockUpdate(@NotNull Location loc, @NotNull TileState tileState) throws IllegalArgumentException
+	{
+		Preconditions.checkNotNull(loc);
+		Preconditions.checkNotNull(tileState);
+		//Pretend we sent block update
+	}
```

Comment:
Can you please add a Test checking that both the Preconditions throw an exceptioon when provided with a null Value and a Test that checks that no exception is thrown when both values are non-null